packages/static-build/test/build-fixtures/09-build-output-v3/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Is this lock file necessary?

Reply:
They were generated when running tests. We can either keep them or .gitignore them. They are not necessary, no. Maybe gitignore is best, then.